scripts/jenkins-yoctobuild-build.sh

```diff
@@ -83,6 +83,17 @@ EOF
          "$PR_STATUS_ENDPOINT"
 }
 
+
+board_support_update() {
+    BOARD=$1
+    STATUS=$2
+
+    curl -u "$BS_JENKINS_AUTH" -H "Content-Type: application/json" \
+         -X POST \
+         -d "'{"jenkins_url":"$BUILD_URL", "commit": "$GIT_COMMIT", "branch": "$META_MENDER_REV", "build": "$BUILD_NUMBER", "status": "$STATUS"}'" \
+         -X POST https://board-support.mender.io/board/"$BOARD"/ci_report
```

Comment:
I suggest `|| true` at the end here. I don't think the whole build should fail if this call fails.